scripts/redirect_url_utils.py

```diff
@@ -28,11 +59,30 @@
 </html>'''
 
 def apply_map():
+  _add_redirects_from_file(FILE_TO_URL)
+
+def update_map(file):
+  """Update the main redirects file with the new redirect pairs in 'file'."""
+  if not os.path.isfile(file):
+    print("'%s' is not a file. Try again." % file)
+    return False
+  _add_redirects_from_file(file)
+  with open(FILE_TO_URL, 'r', encoding='utf-8') as f:
+    redirects = f.readlines()
+  with open(file, 'r', encoding='utf-8') as f:
+    new_redirects = f.readlines()
+  for new_re in new_redirects:
+    redirects.append(new_re)
+  with open(FILE_TO_URL, 'w', encoding='utf-8') as w:
+    for r in redirects:
+      w.write(r)
+
+def _add_redirects_from_file(file):
   total_files = 0
-  with open(FILE_TO_URL, encoding="utf-8") as file_to_url:
+  with open(file, encoding="utf-8") as file_to_url:
     cvs_reader = csv.reader(file_to_url, delimiter=',', quotechar='|')
     for row in cvs_reader:
-      file_path = row[0].lower()
+      file_path = row[0]
```

Comment:
Sorry, this should be the last request. We need `.lower()` when redirects are created paths should be lowercase. (not super critical when all folders are in place, but we can get in trouble when redirects created before we have a content from the build)